torch/_refs/__init__.py

```diff
@@ -1849,6 +1850,10 @@ def view(a: TensorLikeType, shape: ShapeType) -> TensorLikeType:
     return _reshape_view_helper(a, shape, allow_copy=False)
 
 
+def ravel(a: TensorLikeType) -> TensorLikeType:
+    return reshape(a, (-1,))
```

Comment:
IIRC, ref.reshape always returns a contiguous copy

Reply:
I don't think so -- it should work just like `torch.reshape` and sometimes return a view